staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -50,11 +50,16 @@ function kube::codegen::internal::git_grep() {
 #   --boilerplate <string = path_to_kube_codegen_boilerplate>
 #     An optional override for the header file to insert into generated files.
 #
+#   --extra-peer-dir <string>
```

Comment:
I don't feel strongly, but this is *so* close to the actual internal flag it is plumbing to... should we just use `--extra-peer-dirs` here as well?

Reply:
I did that on purpose.  The tool's `--extra-peer-dirs` takes `--extra-peer-dirs="foo,bar,bat"` or `--extra-peer-dirs=foo --extra-peer-dirs=bar --extra-peer-dirs=bat` (which I don't love but oh well).  This script does NOT parse the comma-delimited version